lib/fileHelpers.ts

```diff
@@ -0,0 +1,11 @@
+import * as Constants from './CONST';
+
+const isVideoFile = (fileName: string) => {
+    const lowerCaseFileName = fileName.toLowerCase();
+    return Constants.CONST.VIDEO_EXTENSIONS.some((ext) => lowerCaseFileName.endsWith(ext));
+};
+
+export {
+    // eslint-disable-next-line import/prefer-default-export
```

Comment:
Please use a default export here. I think the only time we should really suppress this lint error is when we know the file is going to be updated soon with new methods. I don't think that is the case for this file.